src/main/kotlin/cacophony/codegen/patterns/Pattern.kt

```diff
@@ -7,7 +7,7 @@ import cacophony.controlflow.*
 data class SlotFill(
     val valueFill: Map<ValueLabel, CFGNode.Value>,
```

Comment:
I think this should be `Map<ValueLabel, VirtualRegister>`, but we can also talk about this on tommorow labs